solutions/monitoring_platforms_intro/index.asciidoc

```diff
@@ -0,0 +1,131 @@
+
+** *Health monitoring:* purpose of health monitoring is to generate a snapshot of the current health of the system so that you can verify that all components of the system are functioning as expected. This discipline includes checking for bugs and errors. 
+** *Availability monitoring:* A truly healthy system requires that the components and subsystems that compose the system are available. Availability monitoring is closely related to health monitoring. But whereas health monitoring provides an immediate view of the current health of the system, availability monitoring is concerned with tracking the availability of the system and its components to generate statistics about the uptime of the system.
+** *Performance monitoring:* As the system is placed under more and more stress (by increasing the volume of users), the size of the datasets that these users access grows and the possiblity of failure of one or more components becomes more likely. Frequently, component failure is preceded by a decrease in performance. If you're able detect such a decrease, you can take proactive steps to remedy the situation.
+** *SLA Monitoring:* SLA monitoring is closely related to performance monitoring. But whereas performance monitoring is concerned with ensuring that the system functions *optimally*, SLA monitoring is governed by a contractual obligation that defines what optimally actually means. You can calculate the percentage availability of a service over a period of time by using the following formula: `%Availability =  ((Total Time – Total Downtime) / Total Time ) * 100`
+--
+
+Providing a control plane requires a monitoring pipeline that should be implemented as feedback loop. The pipeline transforms raw telemetry into meaningful information that the monitoring consumer can use to determine the state of the system. The loop ensures that lessons learnt are the starting point for further improvements on the defined service side. E.g. by adaptive scaling depending on monitored traffic. The entire monitoring must be compliant and provide integration features. The conceptual stages of the pipeline are as follows:
+
+* Data Sources/ Instrumention (Monitored resources): concerned with identifying the sources from where the telemetry needs to be captured, determining which data to capture and how to capture it.
+* Collection/ Storage (Monitoring plane)
+* Analysis/ Diagnosis (Monitoring plane): generate meaningful information that an monitoring consumer can use to determine the state of the system
+* Visualization/ Alerting (Monitoring plane): decisions about possible actions to take and then feed the results back into the instrumentation and collection stages
+
+The picture below summarizes the aspects:
+
+image::monitoring.png[Monitoring,width=757px,height=360px]
```

Comment:
I am not sure if it is a good/common way to store the source of the file as powerpoint file. I think it would be better to store it as `.drawio` or `.svg` file as we do it for example in the devon4j repository. Nothing that has to be done necessarily in this PR. Maybe you can add a following task to port the image to svg/drawio format.